sdks/python/apache_beam/internal/dill_pickler.py

```diff
@@ -46,9 +49,30 @@
 
 settings = {'dill_byref': None}
 
-if sys.version_info >= (3, 10) and dill.__version__ == "0.3.1.1":
-  # Let's make dill 0.3.1.1 support Python 3.11.
+# Let's make dill 0.3.1.1 support Python 3.11.
+patch_save_code = sys.version_info >= (3, 10) and dill.__version__ == '0.3.1.1'
 
+
+# MOE:begin_strip
```

Comment:
MOE doesn't make sense for external code; you can make this unconditionally true.

Reply:
Updated